lib/src/features/home/repositories/storage_repository.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:ecommerce_app/src/features/home/repositories/firebase_storage_repository.dart';
+
+class StorageRepository {
```

Comment:
This repo just calls methods from another repo, can we just use FirebaseStorageRepository instead?